dask/dataframe/core.py

```diff
@@ -999,10 +1000,9 @@ def rolling(self, window, min_periods=None, freq=None, center=False,
         """
         from dask.dataframe.rolling import Rolling
 
-        if not isinstance(window, int):
-            raise ValueError('window must be an integer')
-        if window < 0:
-            raise ValueError('window must be >= 0')
+        if isinstance(window, int):
+            if window < 0:
+                raise ValueError('window must be >= 0')
```

Comment:
Essentially yes, with the caveat that if you're using a string, you have to have a DatetimeIndex